tests/keras/optimizers_test.py

```diff
@@ -78,6 +78,11 @@ def test_nadam():
     _test_optimizer(optimizers.Nadam())
 
 
+def test_norm_grads():
+    sgd = optimizers.SGD(lr=0.01, momentum=0.9, normalize_gradients=True)
+    _test_optimizer(sgd)
```

Comment:
I think we should add a correctness test: a test that checks the updated variables have norm one. We are not doing this for the value clipping test, but that's an oversight.